doc/whats_new.rst

```diff
@@ -4317,4 +4319,4 @@ David Huard, Dave Morrill, Ed Schofield, Travis Oliphant, Pearu Peterson.
 
 .. _Manvendra Singh: https://github.com/manu-chroma
 
-.. _Ibraim Ganiev: https://github.com/olologin
\ No newline at end of file
+.. _Ibraim Ganiev: https://github.com/olologin
```

Comment:
i am not sure as well... i didn't intentionally make any changes. maybe a line ending (e.g. CRLF vs LF)? Not sure if that would affect the diff though...